geth/node/syncpoll.go

```diff
@@ -19,13 +18,18 @@ var (
 // SyncPoll provides a structure that allows us to check the status of
 // ethereum node synchronization.
 type SyncPoll struct {
-	downloader *downloader.Downloader
+	downloader Downloader
```

Comment:
- Is this being done for tests reasons?
- Is there amply reasons for not using the `downloader.Downloader` struct?

These are important questions that are needed for answering, if we use interface to map all structs coming from external package, we will face a case where our code does become brittle and one small change in `goethereum` will cause another massive reveberation to status-go code because of these interfaces.

Your thoughts @tiabc ?